Makefile

```diff
@@ -5,13 +5,16 @@ COMMIT_HASH := $(shell git log -1 --pretty=format:"%h")
 version:
 	echo "VERSION='0.8.0-$(COMMIT_HASH)'" > frigate/version.py
 
+web:
+	cd web && npm install && npm run build
```

Comment:
Looks like a few deps are missing to build this:

```
npm run build

> frigate@ build /home/patrick/workspaces/homeassistant/frigate/frigate/web
> snowpack build

[snowpack] ! building source files...
[snowpack] ✘ file:///home/patrick/workspaces/homeassistant/frigate/frigate/web/src/index.css
  Error: Command failed with ENOENT: postcss
spawn postcss ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frigate@ build: `snowpack build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the frigate@ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/patrick/.npm/_logs/2021-01-13T16_48_21_842Z-debug.log
```

Running this resolves the issue:
```
npm install --save-dev @snowpack/plugin-postcss postcss postcss-cli
```

See https://www.npmjs.com/package/@snowpack/plugin-postcss

Reply:
Oh no! I had thought they were direct dependencies, but looks like they're marked as peers. 

https://github.com/blakeblackshear/frigate/pull/518